scss/components/_table-styles.scss

```diff
@@ -253,6 +253,84 @@ h3 + .simple-table {
   td:first-child {
     border-right: 1px solid $gray-medium;
     padding-left: u(4rem);
-    font-style: italic;
+     font-style: italic;
   }
 }
+
+// For custom_table_block in streamfactory
+
+.custom-table {
+  box-shadow: -8px 0px 9px -2px $gray-medium inset;
+  overflow-x: scroll;
+  display: block;
+  margin-bottom: u(1rem);
+
+  table {
+    @extend .simple-table;
+    width: auto;
+
+    th {
+      @extend .simple-table__header;
+      table-layout: auto;
```

Comment:
Also, let's add `line-height: 1;` to tighten up the spacing:
![image](https://cloud.githubusercontent.com/assets/1696495/24434764/6802f328-13e6-11e7-8995-79adfe433664.png)